Docs - add rocPyJPEG (#193)
ROCm/rocPyDecode: docs/install/rocPyDecode-install.rst

.. meta::
  :description: rocPyDecode and rocPyJPEGDecode Installation
  :keywords: install, rocPyDecode, rocPYJPEGDecode, AMD, ROCm, wheel, docker, bare metal

********************************************************************
Installing rocPyDecode and rocPyJPEGDecode
********************************************************************

The rocPyDecode source code and its installation scripts are available from the `rocPyDecode GitHub Repository <https://github.com/ROCm/rocPyDecode>`_. 

rocPyJPEGDecode is installed with rocPyDecode if `rocJPEG <https://rocm.docs.amd.com/projects/rocJPEG/en/latest/index.html>`_ is already installed on the system. If rocJPEG is not already installed, the rocPyJPEGDecode libraries won't be installed.

> The full list of prerequisites for both rocPyDecode and rocPyJPEGDecode can be found in :doc:`rocPyDecode prerequisites <./rocPyDecode-prerequisites>`.

Reviewer: ```suggestion
The full list of prerequisites for both rocPyDecode and rocPyJPEGDecode can be found in :doc:`rocPyDecode and rocPyJPEGDecode prerequisites <./rocPyDecode-prerequisites>`.
```

Author: I've removed all the rocPyJpegDecodes for the most part.

.. meta::
  :description: rocPyDecode Installation
  :keywords: install, rocPyDecode, rocPyJpegDecode, AMD, ROCm, wheel, docker, bare metal

********************************************************************

> Installing rocPyDecode 

Reviewer: in the line below, shouldn't the github icon show up?
![image](https://github.com/user-attachments/assets/70a9bff1-ec2b-4109-9d44-2314cc9c508c)

rocPyJpegDecode is installed with rocPyDecode if `rocJPEG <https://rocm.docs.amd.com/projects/rocJPEG/en/latest/index.html>`_ is already installed on the system. If rocJPEG is not already installed, the rocPyJpegDecode libraries won't be installed.

The full list of prerequisites for both rocPyDecode and rocPyJpegDecode can be found in :doc:`rocPyDecode prerequisites <./rocPyDecode-prerequisites>`.

The develop branch is the default rocPyDecode branch. The develop branch is intended for users who want to preview new features or contribute to the rocPyDecode code base. If you don't intend to preview new features or contribute to the codebase, clone the rocPyDecode branch that corresponds to your version of ROCm.

rocPyDecode can be installed with :doc:`CMake <./rocPyDecode-cmake-install>` or with the :doc:`pip package manager <./rocPyDecode-pip-install>`. 

The CMake installation method can be used to create deb files and zipped tar files for distribution.

The pip installation method can be used to generate Python egg and wheel files for distribution. 

.. note::

>   The deb files, tar files, and wheel files will include rocPyJpegDecode if rocJPEG was installed on the system prior to installing rocPyDecode.

Reviewer: ```suggestion
  The deb files, tar files, and wheel files will include rocPyJpegDecode if rocJPEG was installed on the system before installing rocPyDecode.
```

> 

Reviewer: Reorder the sentences - the intro has CMake first and then PIP
